code/modules/spells/spell_code.dm

```diff
@@ -247,6 +248,8 @@ var/list/spells = typesof(/spell) //needed for the badmin verb for now
 			spawn(overlay_lifespan)
 				qdel(spell)
 				spell = null
+	if(spell_flags & INCLUDEUSER)
```

Comment:
/aoe_turf spells override this, I am all too aware of how all these systems work now